spinners.json

```diff
@@ -1464,5 +1464,143 @@
 			"▰▰▰▰▰▰▰",
 			"▰▱▱▱▱▱▱"
 		]
+	},
+	"dwarfFortress": {
+		"interval": 100,
```

Comment:
The animation feels a bit slow. 80 seems a bit smoother.

Reply:
yep, makes sense.